contributing_to_docs/doc_guidelines.adoc

```diff
@@ -533,6 +533,11 @@ link:https://github.com/openshift/openshift-docs/pull/13878/files#diff-615ba1bf3
 example] of how this is applied.
 
 
+== Verification of your content
+All documentation changes must be verified by a QE team associate before merging. There are exceptions for typo level changes and other negotiated documentation sets and distributions.
```

Comment:
Maybe consider including formatting-only changes into the exception list.

Reply:
+1

Reply:
+1